HPEMobileApp/app/search/search-page.xml

```diff
@@ -1,65 +1,101 @@
+                Use {N} custom components to reuse UI across your mobile app.
+                The MyDrawer custom component defined in "/shared/my-drawer" holds the contents of the app drawer.
+                Learn more about custom components and how they can be used in this documentation article:
+                http://docs.nativescript.org/ui/basics#custom-components
+                -->
+                <myDrawer:MyDrawer selectedPage="Search"/>
+            </StackLayout>
+        </nsDrawer:RadSideDrawer.drawerContent>
+
+        <!--
+        Use the RadSideDrawer.mainContent component to display the content of the page when you have an app drawer.
+        Always wrap content in a layout component.
+        -->
+        <nsDrawer:RadSideDrawer.mainContent>
+            <GridLayout class="page-content" rows="auto, auto, auto, auto, auto, auto, auto, auto, auto" columns="*">
+                <Label class="search-title" row="0" text="Please fill one or more field:" />
+                <GridLayout class="search-item" row="1" columns="auto, *" >
+                    <Label class="search-field-label" text="System Name:" col="0" /> 
+                    <TextField class="page-search-field" text="{{ system_ID }}" hint="system name" col="1" />
+                </GridLayout>
+                <GridLayout class="search-item" row="2" columns="auto, *" >
+                    <Label class="search-field-label" text="Company:" col="0" /> 
+                    <TextField class="page-search-field" text="{{ company }}" hint="company name" col="1" />
+                </GridLayout>
+                <GridLayout class="search-item" row="3" columns="auto, *" >
+                    <Label class="search-field-label" text="Serial #:" col="0" /> 
+                    <TextField class="page-search-field" text="{{ serial_Num }}" hint="serial #" col="1" />
+                </GridLayout>
+                <GridLayout class="search-item" row="4" columns="auto, *" >
+                    <Label class="search-field-label" text="Product Family:" col="0" /> 
+                    <TextField class="page-search-field" text="{{ prod_family }}" hint="e.g. '7000'" col="1" />
+                </GridLayout>
+                <GridLayout class="search-item" row="5" columns="auto, *" >
+                    <Label class="search-field-label" text="Model:" col="0" /> 
+                    <TextField class="page-search-field" text="{{ model }}" hint="e.g. '7200c'" col="1" />
+                </GridLayout>
+                <GridLayout class="search-item" row="6" columns="auto, *" >
+                    <Label class="search-field-label" text="OS Version:" col="0" /> 
+                    <TextField class="page-search-field" text="{{ osVersion }}" hint="e.g. '3.2.2.530'" col="1" />
+                </GridLayout>
+                <GridLayout class="search-item" row="7" columns="auto, 2*, auto, *" >
+                    <Label class="search-field-label" text="Location: Region:" col="0" /> 

```

Comment:
This Label is a little confusing. Either split it up into two different Labels with better spacing or just pick one of the fields to use.